templates/cluster/openstack-standalone-cp/values.schema.json

```diff
@@ -0,0 +1,405 @@
+      "clusterIdentity",
+      "controlplane",
+      "worker"
+    ],
+    "properties": {
+      "controlPlaneNumber": {
+        "description": "The number of control plane nodes",
+        "type": "number",
+        "minimum": 1
+      },
+      "workersNumber": {
+        "description": "The number of worker nodes",
+        "type": "number",
+        "minimum": 1
+      },
+      "clusterNetwork": {
+        "type": "object",
+        "properties": {
+          "pods": {
+            "type": "object",
+            "properties": {
+              "cidrBlocks": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              }
+            }
+          },
+          "services": {
+            "type": "object",
+            "properties": {
+              "cidrBlocks": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "minItems": 1,
+                "uniqueItems": true
+              }
+            }
+          },
+          "serviceDomain": {
+            "type": "string",
+            "description": "The service domain for the cluster"
+          }
+        }
+      },
+      "clusterIdentity": {
+        "type": "object",
+        "description": "OpenStack cluster identity object reference",
+        "required": [
+          "name",
+          "cloudName",
+          "region"
+        ],
+        "properties": {
+          "name": {
+            "description": "Cluster identity object name",
+            "type": "string"
+          },
+          "cloudName": {
+            "description": "Name of the entry in the clouds.yaml file to use",
+            "type": "string"
+          }
+        }
+      },
+      "managedSubnets": {
+        "type": "array",
+        "description": "Subnets managed by OpenStack for the cluster",
+        "items": {
+          "type": "object",
+          "properties": {
+            "cidr": {
+              "type": "string",
+              "description": "CIDR block for the subnet"
+            },
+            "dnsNameservers": {
+              "type": "array",
+              "description": "List of DNS nameservers for the subnet",
+              "items": {
+                "type": "string"
+              }
+            }
+          },
+          "required": ["cidr", "dnsNameservers"]
+        }
+      },
+      "managedSecurityGroups": {
+        "type": "object",
+        "description": "Defines whether OpenStack security groups are managed by the provider or specific rules are provided",
+        "properties": {
+          "allowAllInClusterTraffic": {
+            "type": "boolean",
+            "description": "Allow all traffic within the cluster security groups"
+          }
+        },
+        "required": ["allowAllInClusterTraffic"]
+      },      
+      "network": {
```

Comment:
We don't have a `network` section in the `values.yaml`. Is it a leftover?

Reply:
Yes it was. Cleaned it up.